Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Scene.h

```diff
@@ -276,6 +276,8 @@ namespace AZ
 
             RHI::ShaderInputConstantIndex m_timeInputIndex;
             float m_simulationTime;
+            RHI::ShaderInputConstantIndex m_prevTimeInputIndex;
+            float m_prevSimulationTime;
```

Comment:
You may set both times default to 0 here to avoid using uninitialized variables.

Reply:
Initialized both to 0.0!